editor/state.js

```diff
@@ -549,9 +686,10 @@ export function createReduxStore() {
 		saving,
 		notices,
 		userData,
+		collaborationMode,
 	} ) );
 
-	const enhancers = [ applyMiddleware( refx( effects ) ) ];
+	const enhancers = [ applyMiddleware( refx( effects ) ), applyMiddleware( ReduxThunk, grtcMiddleware ) ];
```

Comment:
This can be merged into a single `applyMiddleware`: `applyMiddleware( refx( effects ), grtcMiddleware )`